### PR TITLE
Add sensors for Spotify

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add sensors for Spotify

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/128785
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/spotify.markdown [hunk 1]
```diff
@@ -152,3 +153,7 @@ The `media_content_id` value can be obtained from the Spotify desktop app by cli
 ## Unsupported devices
 
 - **Sonos**: Although Sonos is a Spotify Connect device, it is not supported by the official Spotify API.
+
+## Sensors
+
+The Spotify provides sensors that display information of the current song playing.
```
Comment (frenck) [inline source/_integrations/spotify.markdown]: We should specify what that entails.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/spotify.markdown (hunk 1 above)
Comment (c0ffeeca7) [inline source/_integrations/spotify.markdown]:
```suggestion
Spotify provides sensors that display information about the song that is currently being played. The following sensors are available:

- **Song acousticness**: Indicates how much the sound is free from electronic modification. 100% indicates it not electronically modified.
- **Song danceability**. In percent. Describes how suitable a track is for dancing based on a combination of musical elements including tempo, rhythm stability, beat strength, and overall regularity. The higher the value, the more danceable.
- **Song energy**. In percent.  A measure of intensity and activity. Typically, energetic tracks feel fast, loud, and noisy. For example, death metal has high energy, while a Bach prelude scores low on the scale. Perceptual features contributing to this attribute include dynamic range, perceived loudness, timbre, onset rate, and general entropy. A higher number means more energetic.
- **Song instrumentalness**: In percent. Describes whether a track contains no vocals. “Ooh” and “aah” sounds are treated as instrumental in this context. Rap or spoken word tracks are clearly “vocal”. The higher the value the more instrumental the song is. 
- **Song key**: The estimated overall key of the track. If no key was detected, the value is unknown. For example, C sharp or E flat.
- **Song liveness**: In percent.  Describes the presence of an audience in the recording. Higher liveness values represent an increased probability that the track was performed live.
- **Song mode**: The modality (major or minor) of a song.
- **Song speeciness**: In percent. Describes the presence of spoken words in a song. The more exclusively speech-like the recording (e.g. talk show, audio book, poetry), the higher the value.
- **Song tempo**: The speed of the piece of music that is currently playing, in beats per minute (bpm).
- **Song time signature**: The time signature (meter) is a notational convention to specify how many beats are in each bar (or measure). For example: 4/4, 6/8.
- **Song valence**. In percent. Tracks with high valence sound more positive (happy, cheerful, euphoric), while tracks with low valence sound more negative (sad, depressed, angry).
```
This is the one that I could see. Please correct me if I missed any.

Comment (joostlek) [reply source/_integrations/spotify.markdown]: I'm going to add more in a followup, will add them and add them here as well

File: source/_integrations/spotify.markdown [hunk 2]
```diff
@@ -152,3 +153,19 @@ The `media_content_id` value can be obtained from the Spotify desktop app by cli
...
+- **Song instrumentalness**: In percent. Describes whether a track contains no vocals. “Ooh” and “aah” sounds are treated as instrumental in this context. Rap or spoken word tracks are clearly “vocal”. The higher the value the more instrumental the song is. 
+- **Song key**: The estimated overall key of the track. If no key was detected, the value is unknown. For example, C sharp or E flat.
+- **Song liveness**: In percent.  Describes the presence of an audience in the recording. Higher liveness values represent an increased probability that the track was performed live.
+- **Song mode**: The modality (major or minor) of a song.
+- **Song speeciness**: In percent. Describes the presence of spoken words in a song. The more exclusively speech-like the recording (e.g. talk show, audio book, poetry), the higher the value.
```
Comment (joostlek) [inline source/_integrations/spotify.markdown]:
```suggestion
- **Song speechiness**: In percent. Describes the presence of spoken words in a song. The more exclusively speech-like the recording (e.g. talk show, audio book, poetry), the higher the value.
```

File: source/_integrations/spotify.markdown [hunk 3]
```diff
@@ -152,3 +153,19 @@ The `media_content_id` value can be obtained from the Spotify desktop app by cli
...
+- **Song instrumentalness**: In percent. Describes whether a track contains no vocals. “Ooh” and “aah” sounds are treated as instrumental in this context. Rap or spoken word tracks are clearly “vocal”. The higher the value the more instrumental the song is. 
+- **Song key**: The estimated overall key of the track. If no key was detected, the value is unknown. For example, C sharp or E flat.
+- **Song liveness**: In percent.  Describes the presence of an audience in the recording. Higher liveness values represent an increased probability that the track was performed live.
+- **Song mode**: The modality (major or minor) of a song.
+- **Song speechiness**: In percent. Describes the presence of spoken words in a song. The more exclusively speech-like the recording (e.g. talk show, audio book, poetry), the higher the value.
```
Comment (c0ffeeca7) [inline source/_integrations/spotify.markdown]:
```suggestion
- **Song speechiness**: In percent. Describes the presence of spoken words in a song. The more exclusively speech-like the recording (for example, talk show, audio book, poetry), the higher the value.
```

**Review by c0ffeeca7 — APPROVED** [review]
Nice addition! Thank you, @joostlek :+1:

**Pull request merged**